pkg/domain/infra/abi/containers_runlabel.go

```diff
@@ -173,6 +173,13 @@ func generateRunlabelCommand(runlabel string, img *libimage.Image, inputName str
 				return ""
 			}
 			return d
+		case "HOME":
+			h, err := os.UserHomeDir()
+			if err != nil {
+				logrus.Warnf("Unable to determine user's home directory, got %s", err)
```

Comment:
the common go error style printing is something like
`"Unable to determine user's home directory: %s", err`